packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/ios/loadingaconfiguration.md

```diff
@@ -0,0 +1,67 @@
+Initialize a configuration by passing the appropriate values to
+`IDXClient.Configuration(issuer:clientId:clientSecret:scopes:redirectUri:)`. You can set
+the values directly in your code or load them from a file. This example shows loading the
+values from a property file in your project.
+
+First, create a property list file, such as `Okta.plist`. Next, add key-value pairs with
+the configuration settings for your Application Integration. In the file below, substitue
+your issuer URL for `{yourIssuerUrl}`, the client ID of your Application Integration for
+`{yourClientId}`, and a URI that launches your app. Set the scopes based on the access
+required by your app.
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>issuer</key>
+    <string>{yourIssuerUrl}</string>
+    <key>clientId</key>
+    <string>{yourClientId}</string>
+    <key>redirectUri</key>
+    <string>com.sample.:/callback</string>
+    <key>scopes</key>
+    <string>openid email profile offline_access</string>
+</dict>
+</plist>
+```
+
+Next, add code to create an `IDXClient.Configuration` instance from the property list file. This example uses a decoder to read the values into an `OktaPlistContent` `struct`.
```

Comment:
Are you sure that's correct? The word `struct` is the syntax keyword for a structure... it's the difference between the keyword `class` and talking about a class.

With that in mind, it's likely one of these... any thoughts/preferences:

- `OktaPlistContent`.
- `OktaPlistContent` `struct`.
- `OktaPlistContent` structure.

Reply:
struct is the keyword name for a structure, a syntactic element (much like class can be both a description and a keyword.) So should this bit be:
1. `OktaPlistContent` `struct`.
2. `OktaPlistContent` structure.
3. `OktaPlistContent`. (i.e. just the name of the object)
4. something else?

Reply:
Some part of this is rooted in the fact I am not a fan of the single unformatted space between `OktaPListContent` and `struct` in this case but aside from that, I think that unless you are quoting code in which struct is being used you're referring to it in a non-specific sense so it doesn't need to be formatted. 

In .NET for example, both object and collection are keywords and object types, but I'm always going to talk about the `Authenticators` collection and the `VerifyEnrollmentAysncResponse` object without formatting them because I'm using them for context \ in a generic way.

Reply:
FYI, you should always code-font anything that's a symbol, no matter how it looks (your comment on `VerifyEnrollmentAsyncResponse`. If you need to use a non-code-font, use an plain text equivalent, such as "(the object for) the asynchronous response for verifying enrollment".

As to the actual comment, I'v changed it the text in to option 2 above.